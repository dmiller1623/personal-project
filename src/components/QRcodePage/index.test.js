import React from 'react';
import QRCodePage from '../QRCodePage';
import { shallow } from 'enzyme';

describe('QrCodePage', () => {
  let wrapper;
  it('should match the snapshot', () => {
    wrapper = shallow(<QRCodePage />);

    expect(wrapper).toMatchSnapshot();
  });
});