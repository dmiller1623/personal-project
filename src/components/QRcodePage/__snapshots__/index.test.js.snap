// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QrCodePage should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <QRCodePage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          QRCODE
</h1>,
        <QRCode
          bgColor="#FFFFFF"
          className="qr-code"
          fgColor="#000000"
          level="L"
          renderAs="canvas"
          size={128}
          value="http://localhost:3000/selectedResources"
/>,
      ],
      "className": "qr-page",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "QRCODE",
        },
        "ref": null,
        "rendered": "QRCODE",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "bgColor": "#FFFFFF",
          "className": "qr-code",
          "fgColor": "#000000",
          "level": "L",
          "renderAs": "canvas",
          "size": 128,
          "value": "http://localhost:3000/selectedResources",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            QRCODE
</h1>,
          <QRCode
            bgColor="#FFFFFF"
            className="qr-code"
            fgColor="#000000"
            level="L"
            renderAs="canvas"
            size={128}
            value="http://localhost:3000/selectedResources"
/>,
        ],
        "className": "qr-page",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "QRCODE",
          },
          "ref": null,
          "rendered": "QRCODE",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "bgColor": "#FFFFFF",
            "className": "qr-code",
            "fgColor": "#000000",
            "level": "L",
            "renderAs": "canvas",
            "size": 128,
            "value": "http://localhost:3000/selectedResources",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
